update/release.go

```diff
@@ -231,7 +234,15 @@ func (s ReleaseSpec) calculateImageUpdates(rc ReleaseContext, candidates []*Cont
 		for _, container := range containers {
 			currentImageID := container.Image
 
-			filteredImages := imageRepos.GetRepoImages(currentImageID.Name).FilterAndSort(policy.PatternAll)
+			tagPattern := policy.PatternAll
+			// Only consider _all_ images if specific image requested and forced
+			if s.ImageSpec == ImageSpecLatest || !s.Force {
```

Comment:
Similar dissonance with the comment vs the condition. Here I think it'd be better to have an `if..else`, since the "all" case is the special one:

```
var tagPattern policy.Pattern
// Use the container's filter unless given a specific image and forced
if s.Force && s.ImageSpec != ImageSpecLatest {
  tagPattern = policyPatternAll
} else {
  .. get it from the resource policy
}
```

Reply:
the `else` won't cover the case where `Get`ting the tag fails, though.